src/utils/cryptanalysis.ts

```
@@ -0,0 +1,62 @@
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+// Hack to get __dirname in both ESM and CJS
+const currentDir = typeof __dirname !== 'undefined'
+  ? __dirname
+  : path.dirname(fileURLToPath(eval('import.meta.url')));
```

Biome (2.4.9)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)